docs/GlutenUsage.md

```diff
@@ -15,8 +15,7 @@ Based on the different environment, there are some parameters can be set via -D
 | build_velox                                           | Enable or Disable building the CPP part of Velox backend                                                                                                                           | OFF                                                |
 | velox_home (only valid when build_velox is ON)        | The path to the compiled Velox project. When building Gluten with Velox, if you have an existing Velox, please set it.                                                             | /PATH_TO_GLUTEN/ep/build-velox/build/velox_ep      |
 | compile_velox(only valid when velox_home is assigned) | recompile exising Velox use custom compile parameters                                                                                                                              | OFF                                                |
-| velox_build_type                                      | The build type Velox was built with from source code. Gluten uses this value to locate the binary path of Velox's binary libraries.                                                | release                                            |
-| debug_build                                           | Whether to generate debug binary library from Gluten's C++ codes.                                                                                                                  | OFF                                                |
+| build_type                              |   The build type Velox was built with from source code. Gluten uses this value to locate the binary path of Velox's binary libraries.                           |  release                                             |
```

Comment:
Let's use prefix of "velox_" on any configurations which only used by velox backend.

Reply:
Yes, build_type -> velox_build_type